src/components/BlogRoll_en.js

```diff
@@ -63,7 +67,8 @@ export default () => (
       }
       allMarkdownRemark(
         sort: { order: DESC, fields: [frontmatter___date] },
-        filter: { frontmatter: { templateKey: { eq: "blog-post" } }}
+        filter: { frontmatter: { templateKey: { eq: "blog-post" },
```

Comment:
i was puzzling how to exclude the posts with different languages, this tricks was the easiest solution for the moment... taken from: https://github.com/angeloocana/angeloocana/blob/4ab106824ca842b7c45e1d347c721cad6db78013/src/pages/blog.en.js#L14

Reply:
but this is refered to the `en` and `it` version see after...